src/utils/fees.js

```diff
@@ -12,7 +12,8 @@ const SEND_FEE_UNITS = {
   SOL: 1000000,
   MATIC: 21000,
   ERC20: 90000,
-  ARBETH: 620000
+  ARBETH: 620000,
+  FUSE: 1000000000
```

Comment:
1000000000 gas for send? Is that correct?

Reply:
I fixed it: https://docs.fuse.io/the-fuse-chain/fuse-token#fees